packages/pigeon/lib/objc_generator.dart

```diff
@@ -413,7 +413,7 @@ class ObjcHeaderGenerator extends StructuredGenerator<ObjcOptions> {
     indent.writeln('@end');
     indent.newln();
     indent.writeln(
-        'extern void ${apiName}Setup(id<FlutterBinaryMessenger> binaryMessenger, NSObject<$apiName> *_Nullable api);');
+        'extern void ${apiName}SetUp(id<FlutterBinaryMessenger> binaryMessenger, NSObject<$apiName> *_Nullable api);');
```

Comment:
It would actually be better as `SetUp${apiName}`, since the function should read as a verb phrase. Sorry, I didn't think about that when I mentioned it the other day.